lstchain/visualization/camera.py

```diff
@@ -111,3 +112,24 @@ def overlay_hillas_major_axis(display, hillas, **kwargs):
                       hillas.y + x * np.sin(hillas.psi.to(u.rad).value),
                       **kwargs,
                       )
+
+
+def display_array_camera(image, camera_geometry, axes=None, **kwargs):
```

Comment:
what is the difference between this and the camera display from ctapipe? In any case this is a very general function that should be implemented there and not in lstchain

Reply:
It was a small helper function that is indeed not very helpful. It simply called the ctapipe CameraDisplay and returned it.
I removed it.